easybuild/easyconfigs/v/Visit/Visit-3.4.1-foss-2023a.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'CMakeMake'
+
+name = 'Visit'
+version = '3.4.1'
+
+homepage = 'https://github.com/visit-dav/visit'
+description = """
+VisIt is an Open Source, interactive, scalable, visualization, animation
+and analysis tool. From Unix, Windows or Mac workstations, users can
+interactively visualize and analyze data ranging in scale from small
+(<101 core) desktop-sized projects to large (>105 core) leadership-class
+computing facility simulation campaigns. Users can quickly generate
+visualizations, animate them through time, manipulate them with a
+variety of operators and mathematical expressions, and save the
+resulting images and animations for presentations. VisIt contains a rich
+set of visualization features to enable users to view a wide variety of
+data including scalar and vector fields defined on two- and
+three-dimensional (2D and 3D) structured, adaptive and unstructured
+meshes. Owing to its customizeable plugin design, VisIt is capabable of
+visualizing data from over 120 different scientific data formats.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/visit-dav/visit/releases/download/v%(version)s/']
+sources = ['%(namelower)s%(version)s.tar.gz']
+checksums = ['942108cb294f4c9584a1628225b0be39c114c7e9e01805fb335d9c0b507689f5']
+
+builddependencies = [('CMake', '3.26.3')]
+dependencies = [
+    ('zlib', '1.2.13'),
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('SciPy-bundle', '2023.07'),
+    ('X11', '20230603'),
+    ('Mesa', '23.1.4'),
+    ('libglvnd', '1.6.0'),
+    ('Qt5', '5.15.10'),
+    ('Qwt', '6.3.0'),
+    ('VTK', '9.2.6'),
```

Comment:
add a comment above to mention that Visit 3.4.1 requires VTK 9.2.x (9.3 doesn't work), ideally with a link that confirms that (maybe https://github.com/visit-dav/visit/issues/19547)